web/libs/editor/src/tags/visual/Collapse.jsx

```diff
@@ -22,76 +23,105 @@ const { Panel } = Collapse;
  * @param {boolean} [accordion=true]  - Works as an accordion
  * @param {string} [bordered=false]   - Shows border
  */
-const PanelModel = types.model({
-  type: "panel",
-
-  _value: types.optional(types.string, ""),
-  value: types.optional(types.string, ""),
-
-  children: Types.unionArray([
-    "view",
-    "header",
-    "labels",
-    "label",
-    "table",
-    "taxonomy",
-    "choices",
-    "choice",
-    "rating",
-    "ranker",
-    "rectangle",
-    "ellipse",
-    "polygon",
-    "keypoint",
-    "brush",
-    "rectanglelabels",
-    "ellipselabels",
-    "polygonlabels",
-    "keypointlabels",
-    "brushlabels",
-    "hypertextlabels",
-    "text",
-    "audio",
-    "image",
-    "hypertext",
-    "audioplus",
-    "list",
-    "dialog",
-    "textarea",
-    "pairwise",
-    "style",
-    "label",
-    "relations",
-    "filter",
-    "timeseries",
-    "timeserieslabels",
-    "paragraphs",
-    "paragraphlabels",
-  ]),
-});
+const PanelModel = types
+  .model({
+    type: "panel",
 
-const Model = types.model({
-  type: "collapse",
+    _value: types.optional(types.string, ""),
+    value: types.optional(types.string, ""),
 
-  size: types.optional(types.string, "4"),
-  style: types.maybeNull(types.string),
+    children: Types.unionArray([
+      "view",
+      "header",
+      "labels",
+      "label",
+      "table",
+      "taxonomy",
+      "choices",
+      "choice",
+      "rating",
+      "ranker",
+      "rectangle",
+      "ellipse",
+      "polygon",
+      "keypoint",
+      "brush",
+      "rectanglelabels",
+      "ellipselabels",
+      "polygonlabels",
+      "keypointlabels",
+      "brushlabels",
+      "hypertextlabels",
+      "text",
+      "audio",
+      "image",
+      "hypertext",
+      "audioplus",
+      "list",
+      "dialog",
+      "textarea",
+      "pairwise",
+      "style",
+      "label",
+      "relations",
+      "filter",
+      "timeseries",
+      "timeserieslabels",
+      "paragraphs",
+      "paragraphlabels",
+    ]),
+  })
+  .views((self) => ({
+    // Indicates that it could exist without information about objects, taskData and regions
+    get isIndependent() {
+      // if value starts with $ it's related to the data but in case of Panel it's just affect title
+      // we may still want to show it even if there is no data
+      // if (self.value && self.value[0] === "$") return false;
```

Comment:
```
      // if (self.value && self.value[0] === "$") return false;
```

Is this meant to be commented out?

Reply:
Yeah... At first I thought that it was a good idea to hide this element of it depends on a data field. But then I figured out that it could be better to display it anyway. This line is  "ready to be uncomented" in case if my last assumption is not right. But I would like to keep it just as an option for a while.

Maybe there is a better way to describe it in the comment but I didn't find it yet. 😅